tools/tools.go

```diff
@@ -0,0 +1,9 @@
+// +build tools
```

Comment:
Can we use below file?
https://github.com/AmitKumarDas/metac/blob/master/hack/tools.go

Reply:
Oh, sorry, I didn't noticed that file existed. I'll update the pr.